CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+This page contains instructions on how to propose and implement feature changes to Presto.
+
+# The Request for Comments
+
+## Overview of the process
+
+```mermaid
+graph TD;
```

Comment:
Curious to know if this `graph TD;` helps in some formatting/rendering? Why is this needed?

Reply:
It makes the graph return **T**op **D**own.